### PR TITLE
fix: top-level ELB error metric and logs, downscale to min capacities

### PR DESCRIPTION
**Pull request by chris13524**
# Description

- Add ELB errors to top of the Grafana like in Notify Server
- Add other ELB metrics to the bottom
- Add error logs from CloudWatch to the top of the Grafana like in Notify Server
- Downscale everything to min-capacities on staging
- Upgrade to latest version of grafonnet-lib
- Switch to grafonnet-lib Redis panels because there was some breaking change
- Disable alarms on ~~prod~~ staging
- Rename `just amigood` to `just devloop` like in other repos e.g. Notify Server

## How Has This Been Tested?

Not tested

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

File: terraform/monitoring/dashboard.jsonnet [hunk 1]
```diff
@@ -52,8 +56,14 @@ dashboard.new(
 )
 
 .addPanels(layout.generate_grid([
-  row.new('ECS'),
+  row.new('Application'),
+    // panels.app.http_request_rate(ds, vars)          { gridPos: pos._4 },
+    // panels.app.http_request_latency(ds, vars)       { gridPos: pos._4 },
```
Comment (chris13524) [inline terraform/monitoring/dashboard.jsonnet:61]: These panels were specific to Notify Server, but leaving them here as a reminder to put request rate and request latency up here at the top as those are important high-level metrics.

Comment (chris13524) [reply terraform/monitoring/dashboard.jsonnet:61]: Opened issue: https://github.com/WalletConnect/blockchain-api/issues/593

File: README.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ curl -X POST "http://localhost:3000/v1?chainId=eip155:1&projectId=someid" --data
 ## Testing
 
 ```bash
-just amigood
+just devloop
```
Comment (nopestack) [inline README.md:50]: why the name change?

**Review by nopestack — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (chris13524) [reply README.md:50]: See PR description

Comment (nopestack) [reply README.md:50]: amigood makes more sense than devloop imo but it's fine

**Review by nopestack — APPROVED** [review]

**Comment by DGBgithub** [comment]
Hey mal eine Frage wer bist du warum kriege ich ständig von dir E-Mails dass ich hier irgendein Konto verbinden soll bei dir ich weiß noch nicht mal wer du bistVon meinem/meiner Galaxy gesendet
-------- Ursprüngliche Nachricht --------Von: Daniel ***@***.***> Datum: 25.03.24  10:51  (GMT+01:00) An: WalletConnect/blockchain-api ***@***.***> Cc: Subscribed ***@***.***> Betreff: Re: [WalletConnect/blockchain-api] fix: top-level ELB error metric
  and logs, downscale to min capacities (PR #592) 
@nopestack approved this pull request.

File: README.md (hunk 2 above)
Comment (chris13524) [reply README.md:50]: The idea was "am I good" to commit but it was awkward to type and the concept of a "developer loop" is more familiar to people.

**Pull request merged**